### PR TITLE
Add codes as per #205

### PR DESCRIPTION
**Pull request by TheDauntless**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #205 .

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 0d26f61c7b966267be4db2bf3c6d630a179d70af: Broken link result: 0 .
  
 Markdown result errorlength: 223 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 94159cb87fa08ce5e1f44bb800729ac169691575: Broken link result: 0 .
  
 Markdown result errorlength: 147 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 6aaad2d9a6fb1c5c06dc352f57bf833511df3712: Broken link result: 0 .
  
 Markdown result errorlength: 147 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 5b43289c71c1845bb798e0b0e80c0dc24ba0151c: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit d96b8d3e53c4a923eec5bc69aa2de2e1c8535578: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by TheDauntless** [comment]
@coky-t Can you please confirm or fix my Japanese translation? :)

**Comment by OMTGreleaser** [comment]
Results for commit d7371f368b3904f461c8934d49432ea8c5546bfd: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by commjoen** [comment]
So category should be “カテゴリー” or “分類”

File: Document/0x10-V5-Network_communication_requirements.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ The purpose of the controls listed in this section is to ensure the confidential
 
 ## Security Verification Requirements
 
-| # | Description | L1 | L2 |
+| # | Code | Description | L1 | L2 |
 | --- | --- | --- | --- |
```
Comment (commjoen) [inline Document/0x10-V5-Network_communication_requirements.md]: i miss a break line here? --- |

File: Document/0x11-V6-Interaction_with_the_environment.md [hunk 2]
```diff
@@ -6,16 +6,16 @@ The controls in this group ensure that the app uses platform APIs and standard c
 
 ## Security Verification Requirements
 
-| # | Description | L1 | L2 |
+| # | Code | Description | L1 | L2 |
 | --- | --- | --- | --- |
```
Comment (commjoen) [inline Document/0x11-V6-Interaction_with_the_environment.md]: missing a --- | here

File: Document/0x12-V7-Code_quality_and_build_setting_requirements.md [hunk 3]
```diff
@@ -6,17 +6,17 @@ The goal of this control is to ensure that basic security coding practices are f
 
 ## Security Verification Requirements
 
-| # | Description | L1 | L2 |
+| # | Code | Description | L1 | L2 |
 | --- | --- | --- | --- |
```
Comment (commjoen) [inline Document/0x12-V7-Code_quality_and_build_setting_requirements.md]: Missing a --- | here

File: Document-de/0x10-V5-Network_communication_requirements.md [hunk 4]
```diff
@@ -6,14 +6,14 @@ Der Zweck dieser Kategorie ist es die Vertraulichkeit und Integrität übertrage
 
 ## Anforderungen
 
-| # | Beschreibung | L1 | L2 |
+| # | Kategorie | Beschreibung | L1 | L2 |
 | --- | --- | --- | --- |
```
Comment (commjoen) [inline Document-de/0x10-V5-Network_communication_requirements.md]: missing a ---- | here

File: Document-de/0x11-V6-Interaction_with_the_environment.md [hunk 5]
```diff
@@ -6,16 +6,16 @@ Die Anforderungen aus dieser Kategorie sollen sicherstellen, dass Plattform-Komp
 
 ## Anforderungen
 
-| # | Beschreibung | L1 | L2 |
+| # | Kategorie | Beschreibung | L1 | L2 |
 | --- | --- | --- | --- |
```
Comment (commjoen) [inline Document-de/0x11-V6-Interaction_with_the_environment.md]: Missing a --- | here

File: Document-de/0x12-V7-Code_quality_and_build_setting_requirements.md [hunk 6]
```diff
@@ -6,18 +6,20 @@ Das Ziel dieser Kategorie ist, sicherzustellen, dass bei der App-Entwicklung Bas
 
 ## Anforderungen
 
-| # | Beschreibung | L1 | L2 |
+| # | Kategorie | Beschreibung | L1 | L2 |
 | --- | --- | --- | --- |
```
Comment (commjoen) [inline Document-de/0x12-V7-Code_quality_and_build_setting_requirements.md]: Missing a --- | here

**Comment by OMTGreleaser** [comment]
Results for commit 50c7c61677d487fb7b82627cc326edb1b18567d3: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by commjoen** [comment]
Will take this one up for now so you can focus on the scripts!

**Comment by OMTGreleaser** [comment]
Results for commit be5dc69cce087fc8e7c85922c30d7f4715256cb5: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit ebe7040400edba931d20f8f836fdbddabaedfbf2: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit d9445aab52e81e712d2ddf33f1c7c3826e940e3e: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit deac879182cc6a7f7d9690b86b7d6c0141571c48: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit 5cf4e329e54a3edce1b98069702c758563c98d19: Broken link result: 0 .
  
 Markdown result errorlength: 7 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 0a8f81c26341b1b214902178399d47c0f60b5f72: Broken link result: 0 .
  
 Markdown result errorlength: 7 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 3607246cb92ef9724719cc225065a5157577a429: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit edac000c18a1d7a49e6e9844298742dc1883a14b: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit 378cbaf774759941ee60c638a348a12ac01789e3: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Pull request merged**